Meshtastic/Helpers/BLEManager.swift

```diff
@@ -848,7 +848,7 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 						connectedHop.time = Date()
 						connectedHop.num = connectedPeripheral.num
 						connectedHop.name = connectedNode.user?.longName ?? "???"
-						connectedHop.snr = Float(routingMessage.snrBack.last ?? 0 / 4)
+						connectedHop.snr = Float(routingMessage.snrBack.last ?? 0) / 4
```

Comment:
Can we code comments about why we are dividing by 4? (if anyone knows the reason)

Reply:
I've updated the comments on what we are doing with the snr fields from the routing message. I've also noted the snr value that represents unknown.